src/IW/App/Error.hs

```diff
@@ -163,20 +176,20 @@ isInvalid _           = False
 -- Helpers
 ----------------------------------------------------------------------------
 
--- | Extract the value from a maybe, throwing the given 'AppError' if
--- the value does not exist
+-- | Extract the value from a @Maybe a@, throwing the given @AppError@ if
```

Comment:
The same about `AppError` here. `'AppError'` creates a haddock link to the data type while `@AppError@` is just the code highlight in the documentation.

Reply:
```suggestion
-- | Extract the value from a 'Maybe', throwing the given @AppError@ if
```